server/paymentProviders/thegivingblock/webhook.js

```diff
@@ -17,23 +19,33 @@ export async function webhook(req) {
     const pledgeId = payload.pledgeId;
     const valueAtDonationTimeUSD = payload.valueAtDonationTimeUSD;
 
-    let order = await models.Order.findOne({ where: { data: { pledgeId } } });
+    const order = await models.Order.findOne({ where: { data: { pledgeId } } });
     if (!order) {
       throw new Error(`Could not find matching order. pledgeId=${pledgeId}`);
     }
 
     // update totalAmount with latest value
-    order = await order.update({ totalAmount: Number(valueAtDonationTimeUSD) * 100, currency: 'USD' });
+    await order.update({ totalAmount: Number(valueAtDonationTimeUSD) * 100, currency: 'USD' });
```

Comment:
You can drop the whole block.

Reply:
I've dropped this. I was wondering if we should still use the `DEPOSIT_TRANSACTION` and maybe update the order to store that fact. I think this transaction acknowledges the fact the the deposit was done properly and we are waiting for the Giving Block to convert it. When we don't use this until the `TRANSACTION_CONVERTED` is received we might not have any guarantee that the giving block is working on the conversion (or if they have received it). 🤔

Reply:
If we have something useful to do we can do it, perhaps updating order.data with the result. In another PR perhaps?

Reply:
One thing I was thinking is perhaps we can show the contributor that the transaction is under processing by the giving block (with some sort of tag or something in the UI). This way they have a bit more information than the `PENDING` status. But maybe we'll keep this in mind for the future as I don't know how useful a feature that is in the initial iteration. 🤔